lib/streamlit/testing/local_script_runner.py

```diff
@@ -96,18 +103,26 @@ def forward_msgs(self) -> list[ForwardMsg]:
     def run(
         self,
         widget_state: WidgetStates | None = None,
-        timeout: float = 3,
+        timeout: float | None = None,
     ) -> ElementTree:
         """Run the script, and parse the output messages for querying
-        and interaction."""
+        and interaction.
+
+        Timeout is in seconds, or None to use the default.
```

Comment:
nit: it might make it a bit clearer to say: 

`or None to use the default timeout of the script runner. `